test/unit/behaviours/remove-image.test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const reqres = require('hof-util-reqres');
+const Behaviour = require('../../../apps/rotm/behaviours/remove-image');
+
+describe('behaviours/remove-image', () => {
+
+  it('exports a function', () => {
+    expect(Behaviour).to.be.a('function');
+  });
+
+  class Base {
+    getValues() {}
+  }
+
+  let req;
+  let request;
+  let res;
+  let instance;
+  let RemoveImage;
+
+  beforeEach(() => {
+    request = reqres.req;
+    req = request();
+    req.sessionModel.unset = sinon.stub();
+    req.get = sinon.stub();
+    res = reqres.res();
+    RemoveImage = Behaviour(Base);
+    instance = new RemoveImage();
+  });
+
+  describe('getValues()', () => {
+    beforeEach(() => {
+      sinon.stub(Base.prototype, 'getValues').yields();
+    });
+
+    afterEach(() => {
+      Base.prototype.getValues.restore();
+    });
+
+    it('removes the image if the referrer is /image', () => {
+      req.get.withArgs('Referrer').returns('localhost:8080/image');
+
+      instance.getValues(req, res, () => {
+        req.sessionModel.unset.should.be.calledWith('image');
+        req.sessionModel.unset.should.be.calledWith('image-preview');
+        req.sessionModel.unset.should.be.calledWith('image-url');
+      });
+    });
+
+    it('does not remove the image if the referrer is not /image', (done) => {
+      req.get.withArgs('Referrer').returns('localhost:8080/add-image');
+
+      instance.getValues(req, res, () => {
+        req.sessionModel.unset.should.not.be.called;
+        done();
+      });
```

Comment:
You need to also test that the `image` property is actually set on the `sessionModel` when it is not "unset". 
I say this because

Reply:
you say this because?

Reply:
if I test that, then wouldn't I need to set a value on the sessionModel?  Then I'd need to get that value from the sessionModel? how would I do that? I'm currently stubbing the sessionModel

Reply:
Also wouldn't this make it a integration test rather than a unit test?

Reply:
My bad, I was thinking this was an acceptance test for a minute.

Reply:
cool, that was confusing the hell out of me